<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JTJToolClass</class>
 <widget class="QMainWindow" name="JTJToolClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>902</width>
    <height>832</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>JTJTool</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="RunBtn">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>731</y>
      <width>93</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>执行</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="CmdEdt">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>730</y>
      <width>331</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="TextB">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>431</width>
      <height>721</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="ShowImg">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>0</y>
      <width>421</width>
      <height>751</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>902</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="jtjtool.qrc"/>
 </resources>
 <connections/>
</ui>
